query-node/mappings/membership.ts

```diff
@@ -32,26 +23,27 @@ export async function members_MemberRegistered(db: DatabaseManager, event: Subst
     id: memberId.toString(),
     rootAccount: accountId.toString(),
     controllerAccount: accountId.toString(),
-    handle: convertBytesToString(handle.unwrapOr(null)),
-    about: convertBytesToString(about.unwrapOr(null)),
-    avatarUri: convertBytesToString(avatarUri.unwrapOr(null)),
+    // Handle is required by the runtime during registration. Lack of it will throw an error
+    handle: bytesToString(handle.unwrap()),
+    about: about.isSome ? bytesToString(about.unwrap()) : undefined,
+    avatarUri: avatarUri.isSome ? bytesToString(avatarUri.unwrap()) : undefined,
```

Comment:
Are we ok with `unwrap`s there, or should there rather be `unwrapOr`? It is an optional parameter in runtime extrinsic.

Reply:
`isSome` checks whether the option is set, so we can then safely use `unwrap()`. I've seen this pattern in https://github.com/polkadot-js/apps

Reply:
That's right. `handle` is missing `isSome` check, maybe we should add it there?

Reply:
There is a comment above explaining this:
```
 Handle is required by the runtime during registration. Lack of it will throw an error.
```
Source: https://github.com/Joystream/joystream/blob/giza/runtime-modules/membership/src/lib.rs#L612